smart_campus/app/templates/app/station_new.html

```diff
@@ -10,15 +10,7 @@
 {% block content %}
   <div class="ui container">
     <form class="ui centered big form" method="POST" enctype="multipart/form-data" id="station_form">
-      {% for field in form %}
-        {% if field.errors %}
-          <div class="ui negative message">
-            {% for error in field.errors %}
-              <li>{{ field.label }}: {{ error|escape }}</li>
-            {% endfor %}
-          </div>
-        {% endif %}
-      {% endfor %}
+      {% include 'marco/form_error_marco.html' %}
```

Comment:
Is this marco only for form?
It seems to me that it could be used elsewhere.

Reply:
Yes, it is only for the form error field passed by the render content.

Reply:
What about other kinds of message?